ci/travis/build-docker.sh

```diff
@@ -84,6 +84,9 @@ if [ "${USE_LIBCXX:-}" = "yes" ]; then
   cmake_extra_flags="${cmake_extra_flags} -DGOOGLE_CLOUD_CPP_USE_LIBCXX=ON"
 fi
 
+# We want ${cmake_extra_flags} and ${CMAKE_FLAGS} to expand as separate
```

Comment:
Similar comment to before.  We should probably be using arrays.

Reply:
I agree for `${cmake_extra_flags}`. But `CMAKE_FLAGS` is a user-passed environment variable.

Reply:
So should we do `${cmake_extra_flags}` then?  I agree that it isn't an issue as we set the contents and there are no special characters, but not needing to call it out is nice.

Reply:
I forgot to address this. Let me fix it.